src/Converter/Feature2Texture.js

```diff
@@ -65,27 +66,37 @@ function drawPoint(ctx, x, y, origin, scale, style = {}) {
     pt.multiply(scale);
 
     ctx.beginPath();
-    ctx.arc(Math.round(pt.x), Math.round(pt.y), Math.round(style.radius) || 3, 0, 2 * Math.PI, false);
+    ctx.arc(Math.round(pt.x), Math.round(pt.y), Math.round(style.radius) || 10, 0, 2 * Math.PI, false);
     ctx.fillStyle = style.fill || 'white';
     ctx.fill();
     ctx.lineWidth = style.lineWidth || 1.0;
     ctx.strokeStyle = style.stroke || 'red';
     ctx.stroke();
 }
 
+const coord = new Coordinates('EPSG:4326', 0, 0, 0);
+
 function drawFeature(ctx, feature, origin, scale, extent, style = {}) {
     let gStyle = style;
+    // Determinate with radius point
+    const px = 0.01;
```

Comment:
`px` needs to be determinate with radius, because if radius increases then the condition is wrong.

Reply:
Damn I copied/pasted it without noticing the comment, my bad ! Fixing it.

Reply:
So this seems complicated to do: it depends a lot on the current projection/view, and I don't think this will simply things. What about allowing only a maximum radius size ? I don't see a case where a point is larger than 30 or 50px, and unless someone displays this point on a globe with a zoom at 0, there will be no artefact. What do you say ?

Reply:
Nervemind, it is done simply in fact, my bad